interfaces/builtin/network_manager.go

```diff
@@ -294,6 +294,24 @@ dbus (receive, send)
     peer=(label=###SLOT_SECURITY_TAGS###),
 `
 
+const networkManagerSendIntrospectionSnippet = `
+# Allow us to introspect the network-manager providing snap
+dbus (send)
+    bus=system
+    interface="org.freedesktop.DBus.Introspectable"
+    member="Introspect"
+    peer=(label=###SLOT_SECURITY_TAGS###),
+`
+
+const networkManagerRecvIntrospectionSnippet = `
```

Comment:
This should have ConnectedSlot somewhere in the name.

Reply:
Done